internal/persistence/driver.go

```diff
@@ -7,10 +7,13 @@ import (
 
 // Various default settings used for non-SQLite DB drivers.
 const (
-	DefaultDialTimeout     = 15 * time.Second
-	DefaultMaxConn         = 50
-	DefaultMaxConnLifetime = time.Hour
-	DefaultMaxIdleConn     = 20
+	DefaultDialTimeout       = 15 * time.Second
```

Comment:
Will driver.go change when users want to add implementations?

Reply:
that `SQLOpenFunc` is not used for pool initialization.. I should add something in its docs. maybe you can elaborate on your thoughts about it offline.